src/ethereum/ethereumTypes.ts

```diff
@@ -173,7 +173,7 @@ export interface L1RollupParams {
   dynamicOverheadMethod: string
 }
 
-export interface CalcL1RollupFeeParams {
+export interface CalcOptimismL1RollupFeeParams {
```

Comment:
Should this be called `CalcOptimismRollupFeeParams`? You got rid of the "L1" part from all the other names.